### ISSUE TITLE
[ecosystem] GitHub integration holes - build spec without .git

### ISSUE THREAD

**Issue by krOoze** [KhronosGroup/Vulkan-Docs#195]
The _Releases_ on Github do not build, because they are missing the .git folder, that is needed for that.
Prefered (easy) resolution: Release only contents of the built **out** directory there instead.

Consequently Download ZIP on Code tab have the same problem.
Prefered (easy) resolution: Have a fallback - if .git folder doesn't exist, print "detached from git" or something like it instead in the document header (The commit hash and timestamp can be a lie anyway. BTW the date is printed in language specific format of the build machine too.) The git branch is useful, so it would be nice if it was preserved - e.g. have it as a txt file for fallback (perhaps as first line in Changelog.txt).

**Comment by oddhack** [comment]
I actually don't know anything about how github "Releases" are generated. Apparently this is something automatic. If someone who is more familiar with it can advise on how to control this I can do something about it, although "doing something about it" might just consist of turning them off entirely as accomodating "Releases" has never been something we've even considered.

**Comment by oddhack** [comment]
N.b. there is a github repo backing khronos.org/vulkan/registry/ but it's not currently public-facing. Making that visible seem like it would accomodate people who want direct access to older specs for whatever reason.

**Comment by krOoze** [comment]
Oh nevermind, those are (regular git) tags, not releases (minor grafical difference to that. I don't know, why they have Tags tab there, if they then show tags in Releases tab anyway...).

Also the source code archives seems to be mandatory there for both Tags and Releases (when did that happen??  [Documentation](https://help.github.com/articles/about-releases/) still shows releases with "binaries" only)

As for "binary" releases they do have limited usability (=for those who want to have the original HTML files e.g. to host them themselves for some reason). I myself would like to have offline HTML version (which I like better at the moment), without having to build it (though I have to modify it to not use cloud MathJax, so it doesn't save much work ATM). The MathJax fails, when I use browsers "Save page".

TLDR: **So only thing to be done is to make sure the build won't fail without the `.git` folder.**

**Comment by oddhack** [comment]
I'll take a look at it as time allows. Basically some additional annoying script logic in the Makefile would be required. Wiring the commit corresponding to the generated spec into those documents is important for our internal process.

**Comment by oddhack** [comment]
This should be fixed in the 1.0.16 spec update.
